### ISSUE TITLE
Control panels

### ISSUE THREAD

**Issue by robgietema** [plone/plone-react#29]
Create all control panels. The rest api doesn't provide most of the calls for this yet.

**Comment by tisto** [comment]
True. Though, we have a registry endpoint that allows setting registry settings. Since all control panels are based on the registry now, the question is if we need specific endpoint or if providing a schema for the control panels is sufficient.

**Comment by robgietema** [comment]
For most I think providing a schema would be enough. There are some custom ones though, like users for example.

**Comment by robgietema** [comment]
All controlpanels using the registry are implemented. Missing controlpanels are:
- Addons
- Discussion
- Theming (not needed for headless)
- Syndication
- TinyMCE (not needed for headless)
- Contentrules
- Content settings
- Dexterity content types
- Users and groups
- HTML filtering
- Errors
- Maintainance
- Configuration Registry
- Resource Registry (not needed for headless)

**Comment by avoinea** [comment]
Any work in progress on this? Especially **Dexterity content types**?

**Comment by tisto** [comment]
@avoinea not really. It is hard to find clients who are willing to pay for control panels. Therefore this is something we have to do via the community or via sprints IMHO. The Dexterity content types control panel is indeed a biggy and it would be great if we could find someone who wants to pick this up. I'd be more than happy to help with advice if needed.

**Comment by avoinea** [comment]
@tisto Then I suppose I'm the right guy at the right place :).

I can take **The Dexterity content types control panel**

Where should I start? I suppose **plone.restapi**.

**Comment by tisto** [comment]
@avelino awesome! Yeah, I guess we need to create the rest api endpoints first, then move on to the frontend implementation. Do you want to create an issue on the plone.restapi issue tracker?

**Comment by avoinea** [comment]
@tisto What about this one https://github.com/plone/plone.restapi/issues/355
